src/normalize/entities.ts

```diff
@@ -198,6 +198,44 @@ MimePart.addMapping({
 	attach: ['attachments', AttachmentsInfo]
 });
 
+const SmimeCertsSubjectRfc822Name = new Entity({
+	_content: 'content'
+});
+
+const SmimeCertsSubjectAltName = new Entity({
+	rfc822Name: ['rfc822Name', SmimeCertsSubjectRfc822Name]
+});
+
+const SmimeCertsIssuedBy = new Entity({
+	c: 'country',
+	cn: 'commonName',
+	emailAddress: 'emailAddress',
+	l: 'locality',
+	o: 'organizationName',
+	st: 'state'
+});
+
+const SmimeCertsIssuedTo = new Entity({
+	c: 'country',
+	cn: 'commonName',
+	emailAddress: 'emailAddress',
```

Comment:
no need to specify if key and value are same

Reply:
Fixed.